src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http3.cs

```diff
@@ -31,6 +31,14 @@ public HttpClientHandlerTest_Http3(ITestOutputHelper output) : base(output)
         {
         }
 
+        private async Task AssertProtocolErrorAsync(long errorCode, Func<Task> task)
+        {
+            HttpRequestException outerEx = await Assert.ThrowsAsync<HttpRequestException>(task);
+            _output.WriteLine(outerEx.InnerException.Message);
+            HttpProtocolException protocolEx = Assert.IsType<HttpProtocolException>(outerEx.InnerException);
```

Comment:
Should it be this instead?
```suggestion
            HttpProtocolException protocolEx = Assert.IsType<HttpProtocolException>(outerEx.InnerException);
            _output.WriteLine(protocolEx.Message);
```

Also, if I'm wrong and it's for diagnostics purposes in case it was not a HttpProtocolException, would it be better to include all the info about exceptions, even the top one? Meaning, should we log ToString instead of just Message

Reply:
I copied it from HttpClientHandlerTest.Http2 implementation, I can change it

Reply:
This was somewhat accidental in my PR, we don't have to print anything, though it's informative for manual validation of `HttpProtocolException`'s message.